loopchain/p2p/grpc_helper/grpc_helper.py

```diff
@@ -12,12 +12,17 @@
 # See the License for the specific language governing permissions and
```

Comment:
You don't need to keep GRPC Helper.
GRPC Helper has some disadvantages such as singleton. And P2P package has already provided abstract level.
So keeping GRPC Helper implies 2 levels of abstraction. It makes us confused.

Reply:
I will consider next phase. thank you.